extensions/wikia/AdEngine/src/ad-unit-builder.js

```diff
@@ -5,10 +5,12 @@ export default class AdUnitBuilder {
 	static build(slot) {
```

Comment:
Outside of ticket scope
It looks like `AdUnitBuilder` was designed for building ad units for video slots, whereas the name does not include that. Am I right?

Reply:
Not exactly. It looks like AdUnitBuilder is more general just for building MEGA-type ad units based on global context settings and part in `extensions/wikia/AdEngine/src/slots.js` allow to overwrite this value in slot context `slots.${adSlot.getSlotName()}.videoAdUnit`. It's definitely can be cleaned up in "MEGA Everywhere" ticket.